recipes/itk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class HelloTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    name = "example"
+    version = "4.13.2"
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def requirements(self):
+        self.requires("itk/{0}".format(self.version))
+		
+    def imports(self):
+        self.copy("*.dll", dst="bin", src="bin")
+        self.copy("*.dylib*", dst="bin", src="lib")
+        self.copy("*.so*", dst="bin", src="lib")
+
+    def test(self):
```

Comment:
Please simplify this to use the `run_environment` parameter and exclude the execution of this test_package in the case of cross compiling, like this:

https://github.com/bincrafters/templates/blob/d331c6389ab1e0f898db57d252aa22f15a5e108d/default/test_package/conanfile.py#L15